Essentials/src/main/resources/messages_en.properties

```diff
@@ -1481,3 +1481,6 @@ years=years
 youAreHealed=§6You have been healed.
 youHaveNewMail=§6You have§c {0} §6messages\! Type §c/mail read§6 to view your mail.
 xmppNotConfigured=XMPP is not configured properly. If you do not know what XMPP is, you may wish to remove the EssentialsXXMPP plugin from your server.
+playtimeCommandDescription=Shows a player''s playtime
+playtimeCommandUsage=/<command> [player]
+playtime=§6{0}§6''s Playtime\:§r {1}
```

Comment:
Do not add messages to this file, add them to `messages.properties`, not `messages_en.properties`.
```suggestion
```